cmd/mailbox-controller/main.go

```diff
@@ -95,54 +91,58 @@ func main() {
 		}
 	}()
 
-	// create edgeSharedInformerFactory
-	espwRestConfig, err := espwClientOpts.ToRESTConfig()
+	// create space-aware client
+	spaceManagementConfig, err := spaceMgtOpts.ToRESTConfig()
 	if err != nil {
-		logger.Error(err, "failed to create config from flags")
+		logger.Error(err, "Failed to create space management config from flags")
 		os.Exit(3)
 	}
-
-	edgeClientset, err := edgeclientset.NewForConfig(espwRestConfig)
+	spaceclient, err := spaceclient.NewMultiSpace(ctx, spaceManagementConfig)
 	if err != nil {
-		logger.Error(err, "failed to create clientset for view of edge exports")
-		os.Exit(6)
+		logger.Error(err, "Failed to create space-aware client")
+		os.Exit(4)
```

Comment:
Have you guys ever programmed in BASIC? Or put line numbers in a FORTRAN deck? After a little experience you develop a practice of leaving gaps, for future insertions.

Reply:
Changed

Reply:
done